3rd-party/extensions/hotkeys/js/jquery.fancytree.hotkeys.js

```diff
@@ -15,9 +15,11 @@
 	var initHotkeys = function(tree, data) {
 		$.each(data, function(event, keys) {
 			$.each(keys, function(key, handler) {
-				$(tree.$container).on(event, null, key, function() {
+				$(tree.$container).on(event, null, key, function(evt) {
 					var node = tree.getActiveNode();
 					handler(node);
+					evt.preventDefault();
```

Comment:
Oh, you're correct, `return false` is much simpler and enough. 
I assumed there's no way to control the event object because I couldn't find any event object in the `3rd-party/extensions/hotkeys/js/` nor `jeresig/jquery.hotkeys`.

A single line of comment on `jquery.fancytree.hotkeys.js` or on your example browser will clear this issue and much appreciated. And yes being an explicit plugin option would be good too.